extensions/frontend-tiles/src/graphics-provider/tileset-creators/IModelTileset.ts

```diff
@@ -0,0 +1,104 @@
+ * The tileset is expected to reside at "baseUrl/tileset.json" and to have been produced by the [mesh export service](https://developer.bentley.com/apis/mesh-export/).
+ * If no such tileset exists for the given iModel, return `undefined`.
+ * @see [[FrontendTilesOptions.computeSpatialTilesetBaseUrl]].
+ * @alpha
+ */
+export type ComputeSpatialTilesetBaseUrl = (iModel: IModelConnection) => Promise<URL | undefined>;
+
+/** Arguments supplied to [[CreateIModelTilesetAs3DTiles]]
+ * @alpha
+ */
+export type CreateIModelTilesetAs3DTilesArgs = GetIModelTilesetUrlFromConnectionArgs;
+
+/** Options supplied to [[createIModelTileset]].
+ * @alpha
+ */
+export interface CreateIModelTilesetArgs {
+  /** Provide the base URL for the pre-published tileset for a given iModel.
+   * If omitted, [[obtainMeshExportTilesetUrl]] will be invoked with default arguments, using the access token provided by [[IModelApp]].
+   */
+  computeSpatialTilesetBaseUrl?: ComputeSpatialTilesetBaseUrl;
+  /** The maximum number of levels in the tile tree to skip loading if they do not provide the desired level of detail for the current view.
+   * Default: 4.
+   * Reducing this value will load more intermediate tiles, which causes more gradual refinement: low-resolution tiles will display quickly, followed more gradually by
+   * successively higher-resolution ones.
+   * Increasing the value jumps more directly to tiles of the exact level of detail desired, which may load more, smaller tiles up-front, leaving some areas of the view
+   * vacant for longer; and when zooming out some newly-exposed areas of the view may remain vacant for longer because no lower-resolution tiles are initially available to
+   * fill them. However, tiles close to the viewer (and therefore likely of most interest to them) will refine to an appropriate level of detail more quickly.
+   */
+  maxLevelsToSkip?: number;
+  /** Specifies whether to permit the user to enable visible edges or wireframe mode for batched tiles.
+   * The currently-deployed mesh export service does not produce edges, so this currently defaults to `false` to avoid user confusion.
+   * Set it to `true` if you are loading tiles created with a version of the exporter that does produce edges.
+   * ###TODO delete this option once we deploy an edge-producing version of the exporter to production.
+   * @internal
+   */
+  enableEdges?: boolean;
+  /** Specifies whether to enable a CDN (content delivery network) to access tiles faster.
+   * This option is only used if computeSpatialTilesetBaseUrl is not defined.
+   * @alpha
+   */
+  enableCDN?: boolean;
+  /** Specifies whether to enable an IndexedDB database for use as a local cache.
+  * Requested tiles will then first be search for in the database, and if not found, fetched as normal.
+  * @internal
+  */
+  useIndexedDBCache?: boolean;
+}
+
+/** Global configuration initialized by [[initializeFrontendTiles]].
+ * @internal
+ */
+export const frontendTilesOptions = {
```

Comment:
This was copy-pasted from FrontendTiles.ts. I don't know what happens when you export two variables with the same name from a single package, but I suspect it's not what you intended.

Reply:
This was initially removed from FrontendTiles.ts, but we are now trying to introduce these new providers as alpha without changing how frontend-tiles currently works. So the result is some duplicate code because they have been moved to new files but not removed from the old location.